index.html

```diff
@@ -16,8 +16,19 @@
       },
       {
         name: "Yuri Shkuro",
-        company: "Facebook",
+        company: "Meta",
         w3cid: "104074"
+      },
+      {
+        name: "Daniel Dyla",
+        company: "Dynatrace",
+        companyURL: "https://dynatrace.com",
+        w3cid: "117848"
+      },
+      {
+        name: "J. Kalyana Sundaram",
+        company: "Microsoft",
+        w3cid: "??????"
       }],
       formerEditors:  [{
         name: "Alois Reitbauer",
```

Comment:
I think we can move former editors to the Credits section instead as this will be the first release of the spec. @kalyanaj wdyt?

Reply:
We discussed this in the working group meeting and agreed to keep them in former editors, but remove the company name.